exchange_server/assets/service_checks.json

```diff
@@ -1 +1,33 @@
-[]
+[
+    {
```

Comment:
Wow, we were missing this? You'll need to run the declarative integrations pipeline after this is merged